php_phongo.c

```diff
@@ -2375,8 +2375,19 @@ PHP_MINFO_FUNCTION(mongodb)
 	php_info_print_table_header(2, "mongodb support", "enabled");
 	php_info_print_table_row(2, "mongodb version", MONGODB_VERSION_S);
 	php_info_print_table_row(2, "mongodb stability", MONGODB_STABILITY_S);
+#ifdef HAVE_SYSTEM_LIBMONGOC
+	php_info_print_table_row(2, "libmongoc headers version", MONGOC_VERSION_S);
+	php_info_print_table_row(2, "libmongoc library version", mongoc_get_version());
+#else
```

Comment:
hehe. ok. This is such a minor detail that as long as the output isn't vastly different then it isn't worth spending to much time on this.

One thing that just hit me though, maybe we should update the trace line that reports these versions: 
https://github.com/mongodb-labs/mongo-php-driver-prototype/blob/master/php_phongo.c#L1873

I am hoping the only question we will ever need to ask users for support cases is:

    Can you change your php.ini to include mongodb.debug=/tmp/ and rerun the script as send us the content?

Or

    Can you pass array("debug" => "/tmp/") as 3rd argument to the ctor

So it would be nice to know there if it was bundled/system.
See also: https://jira.mongodb.org/browse/PHPC-442